.github/workflows/build-and-publish.yml

```diff
@@ -21,8 +21,11 @@ jobs:
       with:
         python-version: "3.10"
 
+    - name: Install build dependencies
+      run: pip install hatch
+
     - name: Build the package
-      run: python -m build
+      run: hatch build
```

Comment:
I'd stay with `python -m build` as that is the generic and easiest way (and also recommended by hatch if you don't need any further functions from its command line tool)

Reply:
Oh, i was missing `pip install build` so `python -m build` was not working so i used `hatch build` 🙈 
I'll fix that immediately

Reply:
Oh, hold on, we might want to have hatch for the actual publishing part. :sweat_smile:

Reply:
Ah, the publishing is handled by an action, so no need for hatch